### ISSUE TITLE
When I moved file to another folder, provided a crash 

### ISSUE THREAD

**Issue by tonymacx86PRO** [paul-gauthier/aider#39]
My theory is that when I moved the file into another folder and deleted the folder, aider doesn't updated the repomap and it crashes because the path doesn't exist actually.
```
> /commit Now, I actually moved the Endpoints into the OpenAI namespace.

Commit 6278bce Now, I actually moved the Endpoints into the OpenAI namespace.
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Traceback (most recent call last):
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\Scripts\aider.exe\__main__.py", line 7, in <module>
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\site-packages\aider\main.py", line 293, in main
    coder.run()
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\site-packages\aider\coders\base_coder.py", line 328, in run
    new_user_message = self.run_loop()
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\site-packages\aider\coders\base_coder.py", line 369, in run_loop
    inp = self.io.get_input(
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\site-packages\aider\io.py", line 152, in get_input
    completer_instance = AutoCompleter(root, rel_fnames, addable_rel_fnames, commands)
  File "C:\Users\Coder\AppData\Local\Programs\Python\Python310\lib\site-packages\aider\io.py", line 42, in __init__
    with open(fname, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Coder\\Documents\\Visual Studio Projects\\GigaBrain\\GigaBrain.API\\OpenAI\\Endpoints\\Endpoints.cs'
```
What happened:
I had a file named Endpoint.cs inside Endpoints, i wished to move it into OpenAI root directory of OpenAI API Wrapper, and deleted the Endpoints folder Aider confused because the path doesn't exist at all and it crashes.

Theory how to fix:
Make that every maybe request to aider in TUI will be regenerate the u-ctags and reply after that.

**Comment by paul-gauthier** [comment]
Thanks for reporting this. Just pushed a fixed in commit [5d18ebd](https://github.com/paul-gauthier/aider/commit/5d18ebd7247bb42b20328a1008530f1d252d4513). I linked it to the wrong issue in the commit message unfortunately.

If you install the latest from github, you should pick this up.

**Comment by tonymacx86PRO** [comment]
Okay! Thanks.
